fdpclient/operations.py

```diff
@@ -20,7 +20,10 @@ def create(url, data, format='turtle', **kwargs):
     """
     logger.debug(f'Create metadata on {url} with the content: \n{data}')
     if 'headers' in kwargs:
-        kwargs['headers']['content-type'] = DATA_FORMATS[format]
```

Comment:
I think this worked? Or are you going for symmetry with the else clause?

Reply:
yes It worked. It's also assumed the backend uses `turtle` as default format, just as implemented by us. But maybe other backend implementations use different default format. So here I use the `else` to force it to be `turtle` when unspecified.